= form_with(model: contact, local: true) do |form|
  - if contact.errors.any?
    .notification.is-danger#error_explanation
      p De volgende #{pluralize(contact.errors.count, 'fout', 'fouten')}
         zorgden ervoor dat dit contact niet kon worden aangemaakt:
      .content
        ul
          - contact.errors.full_messages.each do |message|
            li
              = message
  .field
    = form.label 'Naam', class: 'label'
    p.control
      = form.text_field :name, class: 'input'
  .field
    = form.label 'btw-nummer', class: 'label'
    p.control
      = form.text_field :vatnumber, class: 'input'
  = form.fields_for :address, contact.address do |f|
    .field
      = f.label 'Begunstigde', class: 'label'
      = f.text_field :beneficiary, class: 'input'
    .field
      = f.label 'Straat', class: 'label'
      = f.text_field :street, class: 'input'
    .field
      = f.label 'Postcode', class: 'label'
      = f.text_field :zip_code, class: 'input'
    .field
      = f.label 'Stad', class: 'label'
      = f.text_field :city, class: 'input'
    .field
      = f.label 'Land', class: 'label'
      = f.text_field :country, class: 'input'
  .actions.field.is-grouped
    p.control
      = form.submit "Bestendige #{action}", class: 'button is-primary'
    p.control
      = link_to 'Terug', contacts_path, class: 'button is-default'

